src/service/asset_meta_service.rs

```diff
@@ -45,22 +44,22 @@ use super::{
 /// Ties all service code together into a cohesive struct to use for complex operations during the
 /// onboarding and verification processes.
 pub struct AssetMetaService<'a> {
-    /// A wrapper for a [DepsMutC](core::util::aliases::DepsMutC] that allows access to it without
+    /// A wrapper for a [DepsMut](core::util::aliases::DepsMut] that allows access to it without
     /// moving the value.
-    deps: DepsContainer<'a>,
+    deps: Depsontainer<'a>,
```

Comment:
RIP `DepsContainer` lol

Reply:
I have been wondering this whole time if that was an intentional typo, not realizing I was the one who accidentally renamed it 🤦